webscalesqlclient/src

```diff
@@ -1 +1 @@
-webscalesql-5.6/
\ No newline at end of file
+../../build/third-party/webscalesqlclient/webscalesql-5.6
```

Comment:
oh my habit is mkdir build, cd build, then ```cmake .. -DCMAKE_INSTALL_PREFIX=/usr```

so a lot of symlink is not consider about other path location for ***${CMAKE_CURRENT_BINARY_DIR}*** such as build directory!